nautobot/dcim/models/devices.py

```diff
@@ -483,7 +483,7 @@ class Device(PrimaryModel, ConfigContextModel, StatusModel):
     _name = NaturalOrderingField(target_field="name", max_length=100, blank=True, null=True, db_index=True)
     serial = models.CharField(max_length=255, blank=True, verbose_name="Serial number", db_index=True)
     asset_tag = models.CharField(
-        max_length=50,
+        max_length=100,
```

Comment:
Can we make sure that we exercise this increased limit in the UI and API tests?

Reply:
Sorry, I am not quite sure what that means. Do you mean assigning an asset tag of size 100 to test devices?

Reply:
Yeah exactly. Add it to `create_data` in test_api.py (https://github.com/nautobot/nautobot/blob/develop-1.6/nautobot/dcim/tests/test_api.py#L1308) and to `form_data` in test_views.py (https://github.com/nautobot/nautobot/blob/develop-1.6/nautobot/dcim/tests/test_views.py#L1439). Thanks!